--- a/modules/module-mixer-api.c
+++ b/modules/module-mixer-api.c
@@ -72,6 +72,7 @@
 enum {
   SCALE_LINEAR,
   SCALE_CUBIC,
+  SCALE_POWER,
 };
 
 static GType
@@ -81,6 +82,7 @@
   static const GEnumValue values[] = {
     { (gint) SCALE_LINEAR, "SCALE_LINEAR", "linear" },
     { (gint) SCALE_CUBIC, "SCALE_CUBIC", "cubic" },
+    { (gint) SCALE_POWER, "SCALE_POWER", "power" },
     { 0, NULL, NULL }
   };
   if (g_once_init_enter (&gtype_id)) {
@@ -378,6 +380,8 @@
     return 0.0;
   else if (scale == SCALE_CUBIC)
     return cbrt(vol);
+  else if (scale == SCALE_POWER)
+    return pow(vol, 1.0 / 1.8);
   else
     return vol;
 }
@@ -389,6 +393,8 @@
     return 0.0;
   else if (scale == SCALE_CUBIC)
     return vol * vol * vol;
+  else if (scale == SCALE_POWER)
+    return pow(vol, 1.8);
   else
     return vol;
 }
--- a/src/config/wireplumber.conf
+++ b/src/config/wireplumber.conf
@@ -865,7 +865,7 @@
     name = "Default sink volume"
     description = "The default volume for audio sinks"
     type = "float"
-    default = 0.064
+    default = 0.287
     min = 0.0
     max = 1.0
   }
--- a/src/tools/wpctl.c
+++ b/src/tools/wpctl.c
@@ -1880,7 +1880,7 @@
 
   if (--ctl->pending_plugins == 0) {
     g_autoptr (WpPlugin) mixer_api = wp_plugin_find (core, "mixer-api");
-    g_object_set (mixer_api, "scale", 1 /* cubic */, NULL);
+    g_object_set (mixer_api, "scale", 2 /* power */, NULL);
     wp_core_install_object_manager (ctl->core, ctl->om);
   }
 }
